tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/AdminTests.cs

```diff
@@ -395,6 +396,60 @@ public async void TestAddSanitizerThrowsOnBadRecordingId()
             assertion.StatusCode.Equals(HttpStatusCode.BadRequest);
         }
 
+
+        [Fact]
+        public async Task GenerateInstanceThrowsOnBadBodyFormat()
+        {
+            RecordingHandler testRecordingHandler = new RecordingHandler(Directory.GetCurrentDirectory());
+            var httpContext = new DefaultHttpContext();
+            httpContext.Request.Headers["x-abstraction-identifier"] = "UriRegexSanitizer";
+            httpContext.Request.Body = TestHelpers.GenerateStreamRequestBody("{\"value\":\"replacementValue\",\"regex\":[\"sv=2020-08-04&ss=bfqt&srt=sco&sp=rwdlacupitfx&se=2023-08-03T16:52:15Z&st=2021-08-03T08:52:15Z&spr=https&sig=randomsigM%3D\"]}");
```

Comment:
Though we currently do not have CredScan on tool repo, I know the CredScan tool will complain about this. Could you make it not secret like regex or read secret from file?